src/pvrclient-argustv.cpp

```diff
@@ -1223,23 +1223,31 @@ bool cPVRClientArgusTV::_OpenLiveStream(const PVR_CHANNEL &channelinfo)
         case ArgusTV::NoFreeCardFound:
           XBMC->Log(LOG_INFO, "No free tuner found.");
           XBMC->QueueNotification(QUEUE_ERROR, "No free tuner found!");
+          return false;
           break;
```

Comment:
If you `return`, the `break`can go as it's dead code now. Applies to other places in this method.